spot-webdriver/specs/adhoc-meeting.spec.js

```diff
@@ -14,6 +14,19 @@ describe('Can start a meeting', () => {
         expect(spotTV.getMeetingName()).toBe(testMeetingName);
     });
 
+    it('and cancel join', () => {
+        const spotTV = session.getSpotTV();
+        const spotRemote = session.getSpotRemote();
+        const inMeetingPage = spotRemote.getInMeetingPage();
+
+        session.joinMeeting('https://meet.jit.si');
```

Comment:
this locks, because it is not a valid meeting URL? maybe worth leaving a comment?